assets/src/BlockEditorValidation.js

```diff
@@ -49,60 +51,60 @@ export const blockEditorValidation = () => {
     canPublish = currentlyValid;
 
     if (!canPublish) {
-      dispatch( 'core/editor' ).lockPostSaving();
+      dispatch('core/editor').lockPostSaving();
     } else {
-      dispatch( 'core/editor' ).unlockPostSaving();
+      dispatch('core/editor').unlockPostSaving();
     }
   });
 
-  registerPlugin( 'pre-publish-checklist', { render: PrePublishCheckList } );
+  registerPlugin('pre-publish-checklist', {render: PrePublishCheckList});
   wp.hooks.addFilter(
     'blocks.registerBlockType',
     'planet4-plugin-gutenberg-blocks',
     registerAttributeValidations
   );
 };
 
-document.addEventListener('change', e=> {
+document.addEventListener('change', (e) => {
```

Comment:
Have we decided to use `e` instead of `evt`? Or we haven't decided yet?

Reply:
I could not find a rule to enforce this variable name, but we can decide as a team what to do